hydra-node/src/Hydra/Chain/Direct/Wallet.hs

```diff
@@ -188,12 +188,12 @@ getTxId ::
   Ledger.TxId crypto
 getTxId tx = Ledger.TxId $ SafeHash.hashAnnotated (body tx)
 
+-- | This are all the error that can happen during coverFee.
 data ErrCoverFee
-  = ErrNoAvailableUTxO
-  | ErrNotEnoughFunds ChangeError
-  | ErrUnknownInput {input :: TxIn}
-  | ErrNoPaymentUTxOFound
-  | ErrScriptExecutionFailed (RdmrPtr, TransactionScriptFailure StandardCrypto)
+  = ErrNotEnoughFunds ChangeError
+  | ErrNoFuelUTxOFound Text
+  | ErrUnknownInput Text
```

Comment:
Should: Replace `Text` with proper types. Keeping `TxIn` and not explain `ErrNoFuelUTxOFound` is what we want here IMO.